### PR TITLE
operator-sdk bump to v0.18.1

### PR DESCRIPTION
**Pull request by PratikMahajan**
Bumped Operator-SDK to v0.18.1

Shift to use `run packagemanifests` from `run --olm` as `run --olm`
is being deprecated. 

Update client-go and openshift client-go function calls which have 
signature changes related to passing contexts and options.

Updated WNI dependency with latest changes for client-go and 
Openshift client-go function calls.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,43 +6,47 @@ replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503
...
 
 require (
-	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/client-go v0.0.0-20190923180330-3b6373338c9b
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32 // indirect
+	github.com/openshift/api v0.0.0-20200422081840-fdd1b0c14c88
+	github.com/openshift/client-go v0.0.0-20200422192633-6f6c07fc2a70
 	github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer v0.0.0-20200521181434-c9423833fb65
-	github.com/operator-framework/operator-sdk v0.17.0
+	github.com/operator-framework/operator-sdk v0.18.0
```
Comment (aravindhp) [inline go.mod:42]: `v0.18.1`

**Comment by VaishnaviHire** [comment]
LGTM

**Comment by PratikMahajan** [comment]
> LGTM, just have a nit. Let us know if you want someone to go through README.md and validate it once
> 
> Thank you for the PR @PratikMahajan

We are testing using OLM, and not a major change or adding new features. I think existing tests will be sufficient.

**Pull request merged**